src/test/java/org/assertj/core/api/abstract_/AbstractAssert_instanceOf_Test.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * Copyright 2012-2019 the original author or authors.
+ */
+package org.assertj.core.api.abstract_;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.BDDMockito.willReturn;
+import static org.mockito.MockitoAnnotations.initMocks;
+
+import org.assertj.core.api.*;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mock;
+
+/**
+ * Tests for <code>{@link AbstractAssert#instanceOf(InstanceOfAssertFactory)}</code>.
+ */
+public class AbstractAssert_instanceOf_Test extends AbstractAssertBaseTest {
```

Comment:
don't extend `AbstractAssertBaseTest` as it is not relevant for this particular test

Reply:
I was trying to keep the style of the neighbor tests, then I'll get rid of it!